config/default/constants.js

```diff
@@ -20,97 +20,108 @@
  * @namespace constants
  * @memberof config
  * @see Parent: {@link config}
- * @property {number} activeDelegates - The default number of delegates allowed to forge a block.
- * @property {number} maxVotesPerTransaction - The maximum number of votes allowed in transaction type(3) votes.
- * @property {number} blockSlotWindow - The default number of previous blocks to keep in memory.
- * @property {number} blockReceiptTimeOut - Seconds to check if the block is fresh or not.
- * @property {Date} epochTime	- Timestamp indicating the start of lisk core.
- * @property {Object} fees - Object representing amount of fees for different types of transactions.
- * @property {number} fees.send	- Fee for sending a transaction.
- * @property {number} fees.vote - Fee for voting a delegate.
- * @property {number} fees.secondSignature	- Fee for creating a secondSignature.
- * @property {number} fees.delegate - Fee for registering as a delegate.
- * @property {number} fees.multisignature - Fee for multisignature transaction.
- * @property {number} fees.dapp	- Fee for registering as a dapp.
+ * @property {number} ACTIVE_DELEGATES - The default number of delegates allowed to forge a block.
+ * @property {number} ADDITIONAL_DATA.MIN_LENGTH - Additional Data (Min Length)
+ * @property {number} ADDITIONAL_DATA.MAX_LENGTH - Additional Data (Max Length)
+ * @property {number} BLOCK_SLOT_WINDOW - The default number of previous blocks to keep in memory.
+ * @property {number} BLOCK_RECEIPT_TIMEOUT - Seconds to check if the block is fresh or not.
+ * @property {Date} EPOCH_TIME	- Timestamp indicating the start of lisk core.
+ * @property {Object} FEES - Object representing amount of fees for different types of transactions.
+ * @property {number} FEES.SEND	- Fee for sending a transaction.
+ * @property {number} FEES.VOTE - Fee for voting a delegate.
+ * @property {number} FEES.SECOND_SIGNATURE	- Fee for creating a secondSignature.
+ * @property {number} FEES.DELEGATE - Fee for registering as a delegate.
+ * @property {number} FEES.MULTISIGNATURE - Fee for multisignature transaction.
+ * @property {number} FEES.DAPP_REGISTRATION	- Fee for registering as a dapp.
+ * @property {number} FEES.DAPP_WITHDRAWAL	- Fee for registering as a dapp.
```

Comment:
@ManuGowda can you provide descriptions for two constants above, bitte?

Reply:
`DAPP_WITHDRAWAL` I am not sure what should be the description and we dont use that constant in the application except for the get constants function, maybe @4miners can help to explain here.